pom.xml

```diff
@@ -0,0 +1,540 @@
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.github.zentol.flink</groupId>
+		<artifactId>flink-connector-parent</artifactId>
+		<version>1.0</version>
+	</parent>
+
+	<groupId>org.apache.flink</groupId>
+	<artifactId>flink-connector-hbase-parent</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<name>Flink : Connectors : HBase Parent</name>
+	<packaging>pom</packaging>
+	<url>https://flink.apache.org</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/apache/flink-connector-hbase</url>
+		<connection>git@github.com:apache/flink-connector-hbase.git</connection>
+		<developerConnection>scm:git:https://gitbox.apache.org/repos/asf/flink-connector-hbase.git</developerConnection>
+	</scm>
+
+	<properties>
+		<flink.version>1.16.0</flink.version>
+		<flink.shaded.version>15.0</flink.shaded.version>
+
+		<scala.binary.version>2.12</scala.binary.version>
+		<scala.version>2.12.7</scala.version>
+
+		<assertj.version>3.21.0</assertj.version>
+		<hadoop.version>2.8.5</hadoop.version>
+		<hbase1.version>1.4.3</hbase1.version>
+		<hbase2.version>2.2.3</hbase2.version>
+		<httpclient.version>4.5.13</httpclient.version>
+		<httpcore.version>4.4.14</httpcore.version>
+		<jackson.version>2.13.4.20221013</jackson.version>
+		<jsr305.version>1.3.9</jsr305.version>
+		<junit4.version>4.13.2</junit4.version>
+		<junit5.version>5.8.1</junit5.version>
+		<kryo.verison>2.24.0</kryo.verison>
+		<netty.version>4.1.70.Final</netty.version>
+		<zookeeper.version>3.4.14</zookeeper.version>
+
+		<slf4j.version>1.7.36</slf4j.version>
+		<log4j.version>2.17.2</log4j.version>
+
+		<!-- For directory plugin -->
+		<flink.parent.artifactId>flink-connector-hbase-parent</flink.parent.artifactId>
+	</properties>
+
+	<modules>
+		<module>flink-connector-hbase-base</module>
+		<module>flink-connector-hbase-1.4</module>
+		<module>flink-connector-hbase-2.2</module>
+		<module>flink-sql-connector-hbase-1.4</module>
+		<module>flink-sql-connector-hbase-2.2</module>
+	</modules>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-shaded-force-shading</artifactId>
+		</dependency>
```

Comment:
Is this needed?

Reply:
I did this based on the root POM of the `flink-connector-elasticsearch` repo. Now I checked it thoroughly and I think this is needed there because of some spdecific elasticsearch related error, according to the shade plugin config in `flink-connector-parent`. I remove it from here, cause the artifacts have the same content regardless of its presence.